gluonnlp/embedding/token_embedding.py

```diff
@@ -152,25 +152,34 @@ class TokenEmbedding(object):
     Parameters
     ----------
     unknown_token : hashable object or None, default '<unk>'
-        The representation for any unknown token. In other words, any unknown token will be indexed
-        as the same representation.
+        Any unknown token will be replaced by unknown_token and consequently
+        will be indexed as the same representation. Only used if oov_imputer is
+        not specified.
+    init_unknown_vec : callback
+        The callback used to initialize the embedding vector for the unknown
+        token. Only used if `unknown_token` is not None.
+    allow_extend : bool,  default  True
+        If True, embedding vectors for previously unknown words can be added
+        via token_embedding[tokens] = vecs. If False, only vectors for known
+        tokens can be updated.
+    unknown_lookup : object subscriptable with list of tokens returning nd.NDarray, default None
+        If not None, unknown_lookup[tokens] is called for any unknown tokens.
+        The result is cached if unknown_autoextend is True.
+    unknown_autoextend : bool, default True
+        If True, any unknown token for which a vector was looked up in
+        unknown_lookup together with the resulting vector will be added to
+        token_to_idx, idx_to_token and idx_to_vec, adding a new index. This
+        option is ignored if allow_extend is False.
 
-    Attributes
-    ----------
-    idx_to_token : list of strs
-        A list of indexed tokens where the list indices and the token indices are aligned.
-    token_to_idx : dict of strs
-        A dictionary of tokens with their corresponding index numbers; inverse vocab
-    idx_to_vec : mxnet.ndarray.NDArray
-        For all the indexed tokens in this embedding, this NDArray maps each token's index to an
-        embedding vector.
-    unknown_token : hashable object or None
-        The representation for any unknown token. In other words, any unknown token will be indexed
-        as the same representation.
     """
 
-    def __init__(self, unknown_token='<unk>'):
+    def __init__(self, unknown_token='<unk>', init_unknown_vec=nd.zeros, allow_extend=True,
+                 unknown_lookup=None, unknown_autoextend=True):
```

Comment:
set default values so that it's consistent with existing behavior

Reply:
Done